src/demo/app/templates/listado-sidebar/listado-sidebar.html

```diff
@@ -1,7 +1,8 @@
 <plex-layout [aspect]="8">
     <plex-layout-main>
         <plex-title size="lg" titulo="Bienvenido a Plex">
-            <plex-help type="help" titulo="Ayuda de este panel" size="sm" title="este es un atributo [title]"
+            <plex-button type="danger" size="md" routerLink="/templates">Volver</plex-button>
+            <plex-help type="info" titulo="Ayuda de este panel" size="md" title="este es un atributo [title]"
```

Comment:
No se usa más title="" para evitar tooltips repetidos, ahora sólo usamos tooltip=""
```suggestion
            <plex-help type="info" titulo="Ayuda de este panel" size="md" tooltip="este es un atributo [title]"
```